### PR TITLE
fix: use default endpint while route is empty

### LINKED ISSUE #42: Report NPE for table error while creating table
**Describe this problem**

ceresdb server is running as single mode, image: ceresdb/ceresdb-server: nightly-20230223-ce9fa87

Report NPE for table error while creating table

**Steps to reproduce**

**Expected behavior**

**Additional Information**

### PR DESCRIPTION
**Pull request by mrrtree**
# Which issue does this PR close?

Closes #42 

# Rationale for this change
 

# What changes are included in this PR?

Use default endpoint while route is empty.

# Are there any user-facing changes?

no user-facing change

# How does this change test

all unit test paased

### CODE REVIEW

File: ceresdb-protocol/src/main/java/io/ceresdb/RouterClient.java [hunk 1]
```diff
@@ -197,7 +200,6 @@ public CompletableFuture<Map<String, Route>> routeFor(final RequestContext reqCt
 
     public CompletableFuture<Map<String, Route>> routeRefreshFor(final RequestContext reqCtx,
                                                                  final Collection<String> tables) {
-        final long startCall = Clock.defaultClock().getTick();
```
Comment (ZuLiangWang) [inline ceresdb-protocol/src/main/java/io/ceresdb/RouterClient.java:200]: Why remove it? it is used for call time statistics.

File: ceresdb-protocol/src/main/java/io/ceresdb/RouterClient.java [hunk 2]
```diff
@@ -184,6 +184,9 @@ public CompletableFuture<Map<String, Route>> routeFor(final RequestContext reqCt
                         ret = remote;
                     } else {
                         local.putAll(remote);
+                        for (String miss : misses) {
+                            local.putIfAbsent(miss, Route.of(miss, opts.getClusterAddress()));
```
Comment (ZuLiangWang) [inline ceresdb-protocol/src/main/java/io/ceresdb/RouterClient.java:188]: If a table created in a random node, will its routing relationship be updated later?

File: ceresdb-protocol/src/main/java/io/ceresdb/RouterClient.java (hunk 1 above)
Comment (mrrtree) [reply ceresdb-protocol/src/main/java/io/ceresdb/RouterClient.java:200]: some time statistics is not used in current version

File: ceresdb-protocol/src/main/java/io/ceresdb/RouterClient.java (hunk 2 above)
Comment (mrrtree) [reply ceresdb-protocol/src/main/java/io/ceresdb/RouterClient.java:188]: yes, the default route not cached in routeClient

**Review by ZuLiangWang — APPROVED** [review]
LGTM

**Pull request merged**